ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -43,6 +43,14 @@
         (date-format (get @message attr.name) "MMM d, yyyy hh:mm aaa" withTimeZone=true)
       }}
     />
+  {{else if (eq attr.name "link")}}
+    {{#each-in @message.link as |title href|}}
+      <InfoTableRow @label="Link" @value={{or title "None"}}>
```

Comment:
I forgot you're yielding a block here so I don't think this will render "None" if the block is empty 😭

Reply:
Ahh you're right! Putting the old conditional logic back :)